src/plot_utils.py

```diff
@@ -79,7 +79,7 @@ def mi_hist(bdb, generator, col1, col2, num_samples=1000, bins=5):
 
 
 def heatmap(bdb, bql, vmin=None, vmax=None, row_ordering=None,
-        col_ordering=None):
+            col_ordering=None):
```

Comment:
Then can we abide by its style?  I can send you a more global fix to that effect.